src/main/java/net/smartcosmos/dto/metadata/MetadataCreate.java

```diff
@@ -0,0 +1,37 @@
+package net.smartcosmos.dto.metadata;
+
```

Comment:
Swagger is a separate commit, so you can easily uncommit it, if it's not wanted anymore.